libnetwork/sandbox_externalkey_unix.go

```diff
@@ -110,7 +111,7 @@ func processReturn(r io.Reader) error {
 		return fmt.Errorf("failed to read buf in processReturn : %v", err)
 	}
 	if string(buf[0:n]) != success {
-		return fmt.Errorf(string(buf[0:n]))
+		return errors.New(string(buf[0:n]))
```

Comment:
This could also be spelled:
```suggestion
		return fmt.Errorf("%s", buf[0:n])
```